SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -24,6 +24,7 @@
 #include "SpatialLaunchConfigCustomization.h"
 #include "SpatialRuntimeVersionCustomization.h"
 
+#include "../../SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h"
```

Comment:
Is this that include path issue mentioned before? Definitely need a non-path included here.

Reply:
No it wasn't this case, but I'll check if this can be improved.